app/db/repository/variables_repository.rb

```diff
@@ -0,0 +1,24 @@
+module DB
+  module Repository
+    class VariablesRepository
```

Comment:
This should probably be singular `VariableRepository` to match the existing `AuthenticatorRepository`.